### ISSUE TITLE
Export Distro doesn't send to new registered clients

### ISSUE THREAD

**Issue by gkorland** [edgexfoundry/edgex-go#622]
```
POST http://localhost:48071/api/v1/registration
{
..
}
```
Doesn't effect the Export service until the service is restarted

**Comment by jpwhitemn** [comment]
This is even a problem for the rules engine that registers when it bootstraps.  On start of the entire system, the export-distro has to be restarted in order to pick up the bootstrap registrations of services like rules engine.

**Comment by gkorland** [comment]
It seems like a regression in the master I think it used to work in 0.6.1

**Comment by tsconn23** [comment]
Here's the root cause on this.

1.) During the Delhi development cycle, a change was made to drive the numeric portion of the interval at which the client monitor updates itself via configuration instead of a static value. A millisecond value (15000) was put into configuration and the code was not changed accordingly. What had originally been defined like this
`time.Sleep(time.Second * time.Duration(15))`

Became this
`time.Sleep(time.Second * time.Duration(params.Interval))`

 Where params.Interval == 15000.

2.) Since we start all services via Docker with `--consul` enabled, the service clients go out to Consul and look at the registered services for their endpoint information. In the docker-compose, export-client is started before export-distro. When the distro client in the export-client service attempted to load an endpoint from Consul, there was nothing there because export-distro wasn't up yet. Therefore, the distro client was empty and wouldn't refresh for 15000 seconds.

3.) Once that was resolved, I had to eliminate setting the "Path" property on the distro client params struct to "/". Doing so caused faulty URLs to be built like so:
```
NotifyRegistrations url=http://edgex-export-distro:48070//api/v1/notify/registrations
ERROR: 2018/12/06 22:48:00 error from distro: 404 - 404 page not found
```
Notice the double slash after the port number.
